twitter_radix/twitter_radix/pages/home.py

```diff
@@ -0,0 +1,222 @@
+
+from ..components import container
+
+
+def tab_button(name, href):
+    """A tab switcher button."""
+    return rx.link(
+        rx.flex(rx.icon(tag="star"), margin_right="1rem"),
+        name,
+        display="inline-flex",
+        align_items="center",
+        padding="0.75rem",
+        href=href,
+        border="1px solid #eaeaea",
+        font_weight="semibold",
+        border_radius="2rem",
+    )
+
+
+def tabs():
+    """The tab switcher displayed on the left."""
+    return rx.box(
+        rx.vstack(
+            rx.heading("PySocial", size="5"),
+            tab_button("Home", "/"),
+            rx.box(
+                rx.heading("Followers", size="3"),
+                rx.foreach(
+                    HomeState.followers,
+                    lambda follow: rx.vstack(
+                        rx.hstack(
+                            rx.avatar(fallback=follow.follower_username, size="3"),
+                            rx.text(follow.follower_username),
+                        ),
+                        padding="1em",
+                    ),
+                ),
+                padding="2rem",
+                border_radius="0.5rem",
+                border="1px solid #eaeaea",
+            ),
+            rx.button("Sign out", on_click=State.logout, size="3"),
+            align_items="left",
+            gap="4",
+        ),
+        padding_top="2rem",
+        padding_bottom="2rem",
+    )
+
+
+def sidebar(HomeState):
+    """The sidebar displayed on the right."""
+    return rx.vstack(
+        rx.input(
+            on_change=HomeState.set_friend,
+            placeholder="Search users",
+            width="100%",
+        ),
+        rx.foreach(
+            HomeState.search_users,
+            lambda user: rx.vstack(
+                rx.hstack(
+                    rx.avatar(fallback=user.username, size="3"),
+                    rx.text(user.username),
+                    rx.spacer(),
+                    rx.button(
+                        rx.icon(tag="plus"),
+                        on_click=lambda: HomeState.follow_user(user.username),
+                    ),
+                    width="100%",
+                ),
+                padding_top="1rem",
+                padding_bottom="1rem",
+                width="100%",
+            ),
+        ),
+        rx.box(
+            rx.heading("Following", size="3"),
+            rx.foreach(
+                HomeState.following,
+                lambda follow: rx.vstack(
+                    rx.hstack(
+                        rx.avatar(fallback=follow.followed_username, size="3"),
+                        rx.text(follow.followed_username),
+                    ),
+                    padding="1em",
+                ),
+            ),
+            padding="2rem",
+            border_radius="0.5rem",
+            border="1px solid #eaeaea",
+            width="100%",
+        ),
+        align_items="start",
+        gap="4",
```

Comment:
gap => spacing? there's other occurrences in this PR too

Reply:
changed